### PR TITLE
Applied the new POM and Completed the test cases

### PR DESCRIPTION
**Pull request by wardah-sheikh**
- Completed remaining specs

- Applied new POM (Based on junaid bhais feedback)

- URL file contains URLs for the test cases

- Functions file contain all the functions

- Selector file contains all the selectors I have used in automation test cases

- Command.js contains some custom commands

- Fixture files contains the test data

### CODE REVIEW

File: cypress/POM/Cart_page.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+export class Cart{
+    getApparel(){
```
Comment (abrarqa62) [inline cypress/POM/Cart_page.js:3]: Use proper locator here

File: cypress/POM/Checkout_page.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        cy.get('#BillingNewAddress_ZipPostalCode').type(zipcode)
+    }
+    enterPhoneNumber(number){
+        cy.get('#BillingNewAddress_PhoneNumber').type(number)
+    }
+    clickContinue(){
+        cy.get('[name="save"]').contains('Continue').click()
+    }
+    selectShippingmethod(){
+
```
Comment (abrarqa62) [inline cypress/POM/Checkout_page.js]: Why this function is empty, if it is not in use so please remove it.

File: cypress/POM/CompareList_page.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+export class CompareList{
+    clickOnApparel(){
+        cy.contains('Apparel').click()
```
Comment (abrarqa62) [inline cypress/POM/CompareList_page.js:3]: Use proper locator here

File: cypress/POM/Wishlist_page.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+export class Wishlist{
+    clickOnApparel(){
+        cy.contains('Apparel').click()
```
Comment (abrarqa62) [inline cypress/POM/Wishlist_page.js:3]: Same here - Use proper locator here

File: cypress/POM/Cart_page.js (hunk 1 above)
Comment (wardah-sheikh) [reply cypress/POM/Cart_page.js:3]: Acknowledged

**Pull request merged**